agent/src/main/java/com/code_intelligence/jazzer/static_analysis/SootJazzerCFGIdMapper.kt

```diff
@@ -88,6 +88,9 @@ fun <E> DefaultDirectedGraph<Unit, E>.simplify() {
 }
 
 object SootJazzerCFGIdMapper {
```

Comment:
Keeping this around as an `object` means that `outputGraph` will essentially be a global variable that is never reset, which makes it difficult to use the mapper multiple times. 

To fix the build failure, you could instead update the call site at https://github.com/CodeIntelligenceTesting/jazzer/blob/0062eb78986d1a90851935716aaeb598d30f7b17/agent/src/main/java/com/code_intelligence/jazzer/static_analysis/CallGraph.kt#L36.